frappe/www/login.html

```diff
@@ -70,20 +70,29 @@ <h6>{{ _("Or login with") }}</h6>
 </section>
 <section class='for-signup'>
 	<div class="login-content page-card" style="margin-top: 20px;">
-	<form class="form-signin form-signup hide" role="form">
-		<div class="page-card-head">
-			<span class="indicator blue" data-text="{{ _('Sign Up') }}"></span>
+	{%- if not disable_signup -%}
+		<form class="form-signin form-signup hide" role="form">
+			<div class="page-card-head">
+				<span class="indicator blue" data-text="{{ _('Sign Up') }}"></span>
+			</div>
+			<input type="text" id="signup_fullname"
+				class="form-control" placeholder="{{ _('Full Name') }}" required autofocus>
+			<input type="email" id="signup_email"
+				class="form-control" placeholder="{{ _('Email Address') }}" required>
+			<button class="btn btn-sm btn-primary btn-block btn-signup" type="submit">{{ _("Sign up") }}</button>
+		</form>
+	{%- else -%}
+		<div class='page-card-head'>
+			<span class='indicator darkgrey'>{{_("Signup Disabled")}}</span>
```

Comment:
Even though darkgrey is mentioned here, it will be reset by `login.js` here

https://github.com/frappe/frappe/blob/7ff1304182e9a7b5eec59d3860e2a7bf63c39435/frappe/templates/includes/login/login.js#L100-L103

We can change the code slightly

```javascript
$('section:not(.signup-disabled) .indicator').each(function() {
	$(this).removeClass().addClass('indicator').addClass('blue')
		.text($(this).attr('data-text'));
});
```

And adding the following in section markup (Leaving this as a suggestion)

```jinja
<section class='for-signup {{ "signup-disabled" if disable_signup else ""  }}'>
```